### PR TITLE
addressed comments

### PR DESCRIPTION
**Pull request by SvenMuller95**

### CODE REVIEW

**Review by JasoonS — COMMENTED** [review]
Cool, looks good. Will find some time to go through this repo in more detail sometime and think about structure etc

File: README.md [hunk 1]
```diff
@@ -51,11 +50,17 @@ These procedures are designed to enable security personnel to identify, mitigate
 ### To be informed immediately
 
 - Jason Smythe - Founder
-    - Contact Details:  <redacted>
+    - Telegram: @Cooolkid
+    - Whatsapp: <redacted>
+    - Discord: Jason | float & wildcards#9999
 - Jonathon Clark - Founder
```
Comment (DenhamPreen) [inline README.md]:
```suggestion
- Jonathan Clark - Founder
```

File: README.md [hunk 2]
```diff
@@ -51,11 +50,17 @@ These procedures are designed to enable security personnel to identify, mitigate
...
-    - Contact Details: <redacted>
+    - Telegram @jonjonclark
+    - Whatsapp: <redacted>
+    - Discord: jonjon | float#2270
 - Denham Preen - Founder:
-    - Contact Details: <redacted>
+    - Telegram: @denhampreen
+    - Whatsapp: <redacted>
+    - Discord: ! Denham | float.capital#5167
```
Comment (DenhamPreen) [inline README.md]: Noice

File: README.md [hunk 3]
```diff
@@ -306,15 +315,15 @@ Whale trader (relative to liquidity in the given market) where there is noticeab
 2. **Consult with advisors** 
 3. **Update UI to reflect any changes - [@Denham Preen ]**
 4. **Communicate to affected users via Discord and/or Twitter - [@Campbell Easton ]**
```
Comment (DenhamPreen) [inline README.md]: Will be nice to link here

File: README.md [hunk 4]
```diff
@@ -338,12 +347,12 @@ Whale trader (relative to liquidity in the given market) where there is noticeab
 2. **Consult with advisors** - [@Jonjon Moose]
 3. **Update UI to reflect any changes -** [@Denham Preen ]
```
Comment (DenhamPreen) [inline README.md]: Will be nice to link here too. gh or twitter links 👍

**Review by DenhamPreen — APPROVED** [review]
Approving but please address nots before merging 🙏

**Pull request merged**